### PR TITLE
Update layout

### LINKED ISSUE #147: general layout
Reorganize the layout from a bigger picture: header, search bar, filter, card area.  
image

### PR DESCRIPTION
**Pull request by hemarina**
fix #147 

- [x] Change structure of layout
- [x] Make sure it works on phone screen and desktop screen

### CODE REVIEW

File: website/src/pages/styles.module.css [hunk 1]
```diff
@@ -109,4 +109,29 @@
...
+}
+
+.filterAndCard {
+  display: flex;
+  flex-direction: row;
+}
+
+.filter {
+  flex-basis: 20%;
+  min-width: 15rem;
```
Comment (vhvb1989) [inline website/src/pages/styles.module.css]: use only `flex: 2`

```suggestion
  flex: 2;
```

File: website/src/pages/styles.module.css [hunk 2]
```diff
@@ -109,4 +109,29 @@
...
+}
+
+.card {
+  flex-basis: 80%;
+  min-width: 15rem;
```
Comment (vhvb1989) [inline website/src/pages/styles.module.css]:
```suggestion
  flex: 8;
```

File: website/src/pages/styles.module.css [hunk 3]
```diff
@@ -109,4 +109,29 @@
 
 .svgIconFavorite {
   margin-left: 1rem;
+}
+
+.filterAndCard {
+  display: flex;
+  flex-direction: row;
```
Comment (vhvb1989) [inline website/src/pages/styles.module.css]: You can remove this. `row` is the default.

**Review by vhvb1989 — COMMENTED** [review]
Some comments about the style properties.

Here's a good guide about working with flex: https://css-tricks.com/understanding-flex-grow-flex-shrink-and-flex-basis/

And this is a nice playground for experimenting with responsive-flex: https://www.w3schools.com/cssref/tryit.php?filename=trycss3_flex_responsive

**Review by wbreza — APPROVED** [review]
This is all good enough from now to get merged into a feature branch.  I think a lot of the CSS/styles will change as we move away from docusaurus into a custom react app.

**Pull request merged**